### PR TITLE
feat(curations): Add VCS info for remaning `commons-logging-api` versions

### PR DESCRIPTION
**Pull request by sschuberth**

### CODE REVIEW

File: curations/Maven/commons-logging/commons-logging-api.yml [hunk 1]
```diff
@@ -1,3 +1,19 @@
+- id: "Maven:commons-logging:commons-logging-api:1.0.3"
+  curations:
+    comment: |-
+      No SCM tag in the POM: https://repo1.maven.org/maven2/commons-logging/commons-logging-api/1.0.3/commons-logging-api-1.0.3.pom
+    vcs:
+      type: "Git"
```
Comment (fviernau) [inline curations/Maven/commons-logging/commons-logging-api.yml:6]: IMO the `type` and `url` should be set for a range of versions or agnostic of the version.

File: curations/Maven/commons-logging/commons-logging-api.yml [hunk 2]
```diff
@@ -1,3 +1,19 @@
...
+      url: "https://github.com/apache/commons-logging.git"
+      revision: "LOGGING_1_0_3"
```
Comment (fviernau) [inline curations/Maven/commons-logging/commons-logging-api.yml:8]: Is ORT's matching algorithm unable to match this tag?

**Review by fviernau — CHANGES_REQUESTED** [review]

File: curations/Maven/commons-logging/commons-logging-api.yml (hunk 2 above)
Comment (sschuberth) [reply curations/Maven/commons-logging/commons-logging-api.yml:8]: Yes, see this test for a similar case:

https://github.com/oss-review-toolkit/ort/blob/631db0fc6b71a71827a0678c5b0a3b41825174bb/utils/ort/src/test/kotlin/UtilsTest.kt#L84

File: curations/Maven/commons-logging/commons-logging-api.yml (hunk 1 above)
Comment (sschuberth) [reply curations/Maven/commons-logging/commons-logging-api.yml:6]: I've extracted these to a version-independent curation now.

**Review by fviernau — APPROVED** [review]

**Pull request merged**